utils/constants.tsx

```diff
@@ -1240,6 +1240,7 @@ export const ZoomSettings = {
 };
 
 export const Constants = {
+    TEST_ID_COUNT: 1,
```

Comment:
(question) What's the impact of this change? Why this value, too (being set to 1)? With the value of 1, the condition in `userCount` above `index >=0 && index < 1` means effectively that userCount is equal to index only for index 0, in all other cases it's -1.

Reply:
https://github.com/mattermost/mattermost-webapp/pull/11328#discussion_r996907261